process/process.js

```diff
@@ -1,10 +1,12 @@
 const GameData = require('./src/GameData');
 
+const gitURL = process.env.ONETECH_PROCESS_GIT_URL || "https://github.com/jasonrohrer/OneLifeData7.git";
```

Comment:
@UncleGus this should mean you just need to pull latest to your fork of Onetech, and set the environment variable to your OneLifeData repo when processing